scripts/workflows/remove_refs.py

```diff
@@ -0,0 +1,25 @@
+"""
+REMOVE ALL REFERENCES IN A WORKFLOW.
+
+This snippet can be used to remove all references in a workflow.
+
+NOTE: use it only if you know what you are doing. References should not be
+removed.
+"""
+
+from invenio_workflows import workflow_object_class
+from invenio_db import db
+
+
+def remove_references(workflow_id):
+    wf = workflow_object_class.get(workflow_id)
+    print 'Workflow {} is currently in position {}'.format(workflow_id,
+                                                           wf.callback_pos)
+    # Note that an empty list is not schema compliant.
+    del wf.data['references']
+    wf.save()
+    db.session.commit()
+
+    res = wf.continue_workflow(delayed=True)
```

Comment:
If we don't put the `continue_next` param, you are going to have a robot upload error after, right?

Reply:
Check this guy https://github.com/inveniosoftware-contrib/invenio-workflows/blob/master/invenio_workflows/api.py#L324

Reply:
"continue_next" seems to be the default, see: https://github.com/inveniosoftware-contrib/invenio-workflows/blob/master/invenio_workflows/api.py#L346
Anyway I will add it to be more explicit